cyclonedx/model/tool.py

```diff
@@ -37,6 +42,12 @@
     from py_serializable import ObjectMetadataLibrary, ViewType
 
 
+@deprecated(
```

Comment:
reverted, as the added statement is not true. the data model itself was never deprecated. it is just the usage in another model, that is deprecated in certain versions of CycloneDX.